webpack.config.js

```diff
@@ -34,6 +30,9 @@ module.exports = {
   },
   devtool: 'source-map',
   plugins: plugins,
+  optimization: {
+    minimize: PROD
```

Comment:
is this needed? just use the `--mode production` or `--mode development` in package.json scripts